addons/survey/views/survey_question_views.xml

```diff
@@ -174,19 +174,19 @@
                                     <field name="matrix_subtype" attrs="{'invisible':[('question_type','not in',['matrix'])],'required':[('question_type','=','matrix')]}"/>
                                 </group>
                             </group>
-                            <group>
-                                <group string="Display" attrs="{'invisible':[('question_type','not in',['simple_choice', 'multiple_choice'])]}">
-                                    <field name="column_nb" string="Number of columns"/>
-                                    <field name="allow_value_image"/>
-                                </group>
-                                <group string="Conditional Display" attrs="{'invisible': [('questions_selection', '=', 'random')]}">
-                                    <field name="is_conditional"/>
-                                    <field name="triggering_question_id"  options="{'no_open': True, 'no_create': True}"
-                                           attrs="{'invisible': [('is_conditional','=', False)], 'required': [('is_conditional','=', True)]}"/>
-                                    <field name="triggering_answer_id" options="{'no_open': True, 'no_create': True}"
-                                           attrs="{'invisible': ['|', ('is_conditional','=', False), ('triggering_question_id','=', False)],
-                                                   'required': [('is_conditional','=', True)]}"/>
-                                </group>
+                            <group string="Display" attrs="{'invisible':[('question_type','=', 'matrix')]}">
+                                <field name="placeholder" attrs="{'invisible': [('question_type', 'not in', ['text_box', 'char_box', 'date', 'datetime', 'numerical_box'])]}"
+                                   placeholder="When set, this text will appear in the field to help participants answer."/>
```

Comment:
Commit message mentions:

> with a field helper that says

But it's not really the field helper (help attribute of the field), it's actually the placeholder... of the placeholder :D